src/components/NcListItem/NcListItem.vue

```diff
@@ -814,11 +814,11 @@ export default {
 	.list-item-content__details {
 		display: flex;
 		flex-direction: column;
-		justify-content: end;
+		justify-content: center;
```

Comment:
how is this related?

Reply:
This makes the component jump if there is a `extra` added while there was already `details`.
E.g. adding a "unread" notification bubble icon will now make it jump a lot.